channel/channel.ml

```diff
@@ -25,6 +25,10 @@ module Make(Flow:V1_LWT.FLOW) = struct
   type +'a io = 'a Lwt.t
   type 'a io_stream = 'a Lwt_stream.t
 
+  exception End_of_file (* at least one user understands this exception *)
```

Comment:
Not sure what this comment means -- do you mean one external user is already expecting this exception to be thrown?